core/domain/exp_services.py

```diff
@@ -830,6 +837,7 @@ def _create_exploration(
     # but the creation of an exploration object will fail.
     exploration.validate()
     rights_manager.create_new_exploration_rights(exploration.id, committer_id)
+    exploration = classifier_services.train(exploration)
```

Comment:
No, I was asking about why we need to train a classifier when the exploration is created?